### PR TITLE
feat(webex-core): enable validateDomains by default

### PR DESCRIPTION
**Pull request by adamweeks**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-137880

By default, domains authentication is disabled and requires a specific configuration on the JS SDK setting validateDomains to true. This setting should be defaulted to true so that services can spin up before being added to the U2C catalog. 

This request was made as a suggestion by Sunil Veluvali while researching issues around SPARK-136704.

### CODE REVIEW

File: packages/node_modules/@webex/webex-core/src/config.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default {
       hydra: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1',
       hydraServiceUrl: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1'
     },
-    validateDomains: process.env.VALIDATE_DOMAINS === 'true' || false,
+    validateDomains: true,
```
Comment (InteractiveTimmy) [inline packages/node_modules/@webex/webex-core/src/config.js:68]: Don't need this one, see [this case](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-135715).

Comment (adamweeks) [reply packages/node_modules/@webex/webex-core/src/config.js:68]: The line can be removed completely?

Comment (InteractiveTimmy) [reply packages/node_modules/@webex/webex-core/src/config.js:66]: Yes, as all of these values are no longer used by the device plugin. There's a task to remove this information in our backlog.

Comment (lalli-flores) [reply packages/node_modules/@webex/webex-core/src/config.js:66]: Can you link the task?

**Pull request merged**